Prepare for `v2` branch release (#2863)
DataDog/datadog-api-client-typescript: services/cloud_cost_management/CHANGELOG.md

# CHANGELOG

## 0.1.0/2025-10-02

### Changed

> * Add tag pipeline, custom allocation rule and get cloud account by id for AWS/Azure/GCP configs APIs [#2785](https://github.com/DataDog/datadog-api-client-typescript/pull/2785)

Reviewer: ```suggestion
* Add tag pipelines, custom allocation rules, and cloud account retrieval by ID to the AWS, Azure, and GCP configs APIs [#2785](https://github.com/DataDog/datadog-api-client-typescript/pull/2785)
```
